pkg/validator/messages/messages.go

```diff
@@ -96,4 +96,12 @@ const (
 	HostNetworkFailure = "Host network should not be configured"
 	// HostNetworkSuccess message
 	HostNetworkSuccess = "Host network is not configured"
+	// RegistryWhitelistFailure message
+	RegistryWhitelistFailure = "The image registry must match the allowed registries list"
```

Comment:
Nit: Can we add the verbiage of `... allowed registry whitelist...` just so the term whitelist is in this failure? (not a huge deal though)

Reply:
yes. :)